tools/breaking-changes/generate-doc.ts

```diff
@@ -128,17 +160,29 @@ function getMembersDoc(apiElement: any): string {
           doc += `
 
 Previous version: 
-${MD_CODEBLOCK}${memberBreakingChange.previousStateDoc}${MD_CODEBLOCK}
+${MD_CODEBLOCK}${common.getMemberStateDoc(
+            memberBreakingChange.old
+          )}${MD_CODEBLOCK}
 
 Current version: 
-${MD_CODEBLOCK}${memberBreakingChange.currentStateDoc}${MD_CODEBLOCK}
+${MD_CODEBLOCK}${common.getMemberStateDoc(
+            memberBreakingChange.new
+          )}${MD_CODEBLOCK}
 `;
-
+          if (!!memberBreakingChange.migrationComment) {
+            doc += `\n${memberBreakingChange.migrationComment}\n`;
+          }
           break;
         }
         case 'DELETED': {
-          doc += `\n### ${memberBreakingChange.deletedMember.kind} ${memberBreakingChange.deletedMember.name} is removed.\n`;
-          doc += `\n${memberBreakingChange.migrationComment}\n`;
+          const memberMigrationComment = common.findDeletedMemberComment(
+            apiElement,
+            memberBreakingChange.changeElementName,
+            deletedMembersCommentData
+          );
+
+          doc += `\n### ${memberBreakingChange.old.kind} ${memberBreakingChange.old.name} is removed.\n`;
+          doc += `\n${memberMigrationComment}\n`;
```

Comment:
what do you think about prefixing these 'additional' comments as shown below. 

(Valid to all instances in this generated doc automation - reason why I'm asking for the change is because it gets appended below the generated 'automatic' comments, but then there's this random text that appears below, which is weird (in my opinion))
```suggestion
          doc += `\nAdditional Comments: ${memberMigrationComment}\n`;
```

Reply:
Don't have to prefix those in the generated schematics comment though as they're in one line.

Reply:
For the deleted migration comments, I think the title above the comment provides enough context for the migration comment not to look randomly placed:

```
### Method removeEntry is removed.

The method is not used anymore.  Use that other method instead.
```

With the prefix it would look like this:
```
### Method removeEntry is removed.

Additional comment: The method is not used anymore.  Use that other method instead.
```

In the end, it's not a change that is long to implement, but let's discuss what we want to do about it.